### DISCUSSION TITLE
go-redis suport redis7.0

### DISCUSSION THREAD

**Discussion by 631086083** [redis/go-redis#2018] [Ideas]
redis7 has been released, does the community have some  idea or work to do?

**Comment by tanxiao1990** [comment]
why github.com/go-redis/redis/v9 is not surport for redis6 ? it is only surport redis7 ？

**Reply by Odysseusailoon** [reply]
because of resp3? not sure.

**Reply by tanxiao1990** [reply]
> because of resp3? not sure.

failed to ping redis: ERR unknown command `hello`, with args beginning with: `3`, `auth`, `default`, `123456`,

**Reply by vmihailenco** [reply]
Should be fixed in v9.0.0-beta.2
